code/mathematical_algorithms/src/newman_conway/Newman_Conway_Sequence.cpp

```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+#include <vector>
+
+int nthNewmanConwayNumber(int number)
+{
+    std::vector<int> arr;
+    arr.push_back(0);
+    arr.push_back(1);
+    arr.push_back(1);
+    for(size_t i = 3; i <= number; ++i)
+        arr.push_back(arr[arr[i-1]] + arr[i - arr[i-1]]);
```

Comment:
Could you give spaces between operands and operators?

Reply:
Yeah sorry